pdr_backend/lake/test/test_etl.py

```diff
@@ -132,28 +134,28 @@ def test_etl_do_bronze_step(
 
     # Assert payout came from payouts
     assert round(bronze_pdr_predictions_df["payout"][1], 3) == round(
-        _gql_datafactory_etl_payouts_df["payout"][2], 3
+        _gql_datafactory_etl_payouts_df["payout"][1], 3
     )
     assert round(bronze_pdr_predictions_df["payout"][2], 3) == round(
-        _gql_datafactory_etl_payouts_df["payout"][3], 3
+        _gql_datafactory_etl_payouts_df["payout"][2], 3
     )
 
     # Assert stake in the bronze_table came from payouts
     assert round(bronze_pdr_predictions_df["stake"][1], 3) == round(
-        _gql_datafactory_etl_payouts_df["stake"][2], 3
+        _gql_datafactory_etl_payouts_df["stake"][1], 3
     )
     assert round(bronze_pdr_predictions_df["stake"][2], 3) == round(
-        _gql_datafactory_etl_payouts_df["stake"][3], 3
+        _gql_datafactory_etl_payouts_df["stake"][2], 3
     )
 
     # Assert bronze slots table is building correctly
-    table_name = get_table_name(bronze_pdr_slots_table_name)
-    bronze_pdr_slots_records = pds.query_data("SELECT * FROM {}".format(table_name))
+    # table_name = get_table_name(bronze_pdr_slots_table_name)
+    # bronze_pdr_slots_records = pds.query_data("SELECT * FROM {}".format(table_name))
```

Comment:
this doesnt look good/correct

Reply:
this was done do disable bronze_slots_table, check first point from PR description

Reply:
I thought you were trying to get that running here.. but I see now what is going on... I'll see what I can do to help out here so we can prepare the duckdb pr.